plugins/azure/postgresqlserver/flexibleServerVersion.js

```diff
@@ -0,0 +1,63 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'PostgreSQL Flexible Server Version',
+    category: 'PostgreSQL Server',
+    domain: 'Databases',
+    description: 'Ensure PostgreSQL flexible servers is using the latest server version.',
+    more_info: 'The latest version of PostgreSQL for flexible servers will give access to new software features, resolve reported bugs through security patches, and improve compatibility with other applications and services.',
+    recommended_action: 'Upgrade the version of PostgreSQL flexible server to the latest available version..',
+    link: 'https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-supported-versions',
+    apis: ['servers:listPostgresFlexibleServer'],   
+    settings: {
+        server_desired_version: {
+            name: 'Postgresql Flexible Server Desired Version',
+            description: 'Desire Postgresql Flexible Server Version ',
+            regex: '^[0-9]+$',
+            default: '11'
+        }
+    },
```

Comment:
```suggestion
    settings: {
        server_desired_version: {
            name: 'Postgresql Flexible Server Desired Version',
            description: 'Desire Postgresql Flexible Server Version ',
            regex: '^[0-9]+$',
            default: '11'
        }
    },
```
this shouldn't be a setting, we should find the minimum version which should be used and check against it. This is one of those periodic refactoring plugins where we keep checking for deprecated and outdated versions and update them accordingly (also add it to the periodic refactoring sheet).